src/tasks/ts/tokens.ts

```diff
@@ -21,6 +21,7 @@ export interface Erc20Token {
 export interface NativeToken {
   symbol: string;
   decimals: number;
+  wrappedToken: string | null;
```

Comment:
It's a shame that this has to be nullable (I assume we cannot easily deploy WETH on hardhat). If it's optional what is it used for (I can't see any read operation of this value in this PR)?

Reply:
Nice observation! It's indeed unused, both here and in other PRs.

It was originally used to implement `usdValue` in the case where the token is ETH. In the end I concluded that the extra logic was not worth it and simply used WETH instead of ETH in the only place where I needed the price of ETH ([here](https://github.com/gnosis/gp-v2-contracts/blob/7dbc21f510bb31d6b1d5400e0ef67c2252d8a923/src/tasks/withdraw.ts#L218-L223)). However I forgot that this extra field was added for this purpose when releasing this PR.